File: t/VB/DicomAnonymizerDemo/UI/Controls/DataGridViewDisableButtonColumn.vb
' *************************************************************
' Copyright (c) 1991-2019 LEAD Technologies, Inc.              
' All Rights Reserved.                                         
' *************************************************************
Imports System
Imports System.Collections.Generic
Imports System.Linq
Imports System.Text
Imports System.Windows.Forms

Namespace DicomAnonymizer.UI.Controls
   Public Class DataGridViewDisableButtonColumn : Inherits DataGridViewButtonColumn
      Public Sub New()
         Me.CellTemplate = New DataGridViewDisableButtonCell()
      End Sub
   End Class
End Namespace

File: t/CS/DicomAnonymizerDemo/UI/Controls/DataGridViewDisableButtonColumn .cs
// *************************************************************
// Copyright (c) 1991-2019 LEAD Technologies, Inc.              
// All Rights Reserved.                                         
// *************************************************************
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Windows.Forms;

namespace DicomAnonymizer.UI.Controls
{
   public class DataGridViewDisableButtonColumn : DataGridViewButtonColumn
   {
      public DataGridViewDisableButtonColumn()
      {
         this.CellTemplate = new DataGridViewDisableButtonCell();
      }
   }
}
